backend/maelstro/main.py

```diff
@@ -0,0 +1,103 @@
+"""
+Main backend app setup
+"""
+
+from typing import Annotated
+from fastapi import FastAPI, Request, Response, Header
+
+
+app = FastAPI(root_path="/maelstrob")
```

Comment:
can we consider renaming the path to `maelstro-backend` ?

Reply:
:+1:  seems clearer